components/input/gamepad/gamepad_mappings_linux.go

```diff
@@ -4,14 +4,18 @@
 package gamepad
 
 import (
+	"strings"
+
 	"github.com/viamrobotics/evdev"
 
 	"go.viam.com/rdk/components/input"
 )
 
-// GamepadMappings contains all the axes/button translations for each model.
+const stadiaPrefix = "Stadia"
+
+// gamepadMappings contains all the axes/button translations for each model.
 // use evtest on linux figure out what maps to what.
-var GamepadMappings = map[string]Mapping{
```

Comment:
Just an FYI, I originally made this public as I figured someone could inject their own mappings in an init() or early main() code back when we were building custom servers for everything. I don't think that paradigm holds today though, so totally fine with this change.

Reply:
Sounds good, thanks for the info.